CMakeLists.txt

```diff
@@ -445,11 +445,13 @@ psibase_package(
         WASM ${CMAKE_CURRENT_BINARY_DIR}/AuthSig.wasm
         DATA ${Plugins_OUTPUT_FILE_auth_sig} /plugin.wasm
         SERVER r-auth-sig
+        DATA GLOB ${CMAKE_CURRENT_SOURCE_DIR}/services/system/AuthSig/ui/dist/* /
     SERVICE r-auth-sig
         WASM ${CMAKE_CURRENT_BINARY_DIR}/RAuthSig.wasm
     SERVICE verify-sig
         WASM ${CMAKE_CURRENT_BINARY_DIR}/VerifySig.wasm
         FLAGS isAuthService
+    DEPENDS ${AuthSig_js_DEP} ${CMAKE_CURRENT_SOURCE_DIR}/services/system/AuthSig/ui/dist/index.html
```

Comment:
Don't need to add this index.html file if you're already adding `AuthSig_js_DEP`. Although given the CI/CD error, I think the `AuthSig_js` target has yet to be added.